torch/testing/_internal/common_utils.py

```diff
@@ -93,6 +93,18 @@
 DISABLED_TESTS_FILE = '.pytorch-disabled-tests.json'
 SLOW_TESTS_FILE = '.pytorch-slow-tests.json'
 
+disabled_tests_dict = {}
+slow_tests_dict = {}
+
+if os.getenv("IMPORT_SLOW_TESTS", ""):
```

Comment:
So my understand here is that we will just load the JSON structures from files instead of using env variable.  So, there are two parts that I'm trying to understand:

* `IMPORT_SLOW_TESTS = args.import_slow_tests` is a parameter.  I think you mean to use `SLOW_TESTS_DICT` here.  We all get confused looking at these similar name variables.

* Originally, the code use `with open(IMPORT_SLOW_TESTS, 'r') as fp`. This implies that the filename is passed to run_tests.py using `--import-slow-tests MY_SLOW_TEST_FILE.json --import-disabled-tests MY_DISABLED_TEST_FILE.json`. SLOW_TESTS_FILE and DISABLED_TESTS_FILE are the default values to include always when nothing is set ([const](https://docs.python.org/3/library/argparse.html#const))

```
parser.add_argument('--import-slow-tests', type=str, nargs='?', const=SLOW_TESTS_FILE)
parser.add_argument('--import-disabled-tests', type=str, nargs='?', const=DISABLED_TESTS_FILE)
```

This PR changes the usage a bit and always read from SLOW_TESTS_FILE and DISABLED_TESTS_FILE.  Is that what you're trying to achieve?  For example, one line 637, the check is `if os.path.exists(IMPORT_SLOW_TESTS)` but then `with open(SLOW_TESTS_FILE, 'r')` is actually opened on the next line. Thus my confusion

Reply:
ah i see what you mean, i renamed a bunch of stuff so hopefully its slightly more clear what everything does now